docs/reference/config.md

```diff
@@ -20,7 +20,8 @@
 | eventRouter |  | *[Enabled](#enabled) |  |
 | harbor |  | *[Harbor](#harbor) |  |
 | hostPrefix | A prefix to be added to VM hostnames. | string | Yes |
-| importConfigs |  | []string |  |
+| importConfigs | Paths to additional config files to be merged into this config | []string |  |
+| ConfigFrom | List of objects containing karina config | [][ImportDirective](#importdirective) |  |
```

Comment:
To avoid a breaking change initially, the original ImportConfig interface is still in place.  My thought was to allow both to exist in parallel for a while in order to allow time to move all existing config files to the new format.  I'll mark that as deprecated in the docs

Reply:
s/ConfigFrom/configFrom/